include/yacx/Logger.hpp

```diff
@@ -109,7 +109,7 @@ class Logger {
     current_loglevel = severity;
     if (current_loglevel <= limit) {
       std::stringstream prefix_ss;
-      prefix_ss << detail::get_color(severity) << get_datetime() << " "
+      prefix_ss /*<< get_color(severity)*/ << get_datetime() << " "
```

Comment:
I do not understand the error here, please help @LukasSiefke

Reply:
Was ist denn die Fehlermeldung?

Reply:
eh so ein hardcore segfault, weil irg wo bei xstring irg eine funktion out of bounds geht oder so